test/upgrade-tests.sh

```diff
@@ -37,48 +37,33 @@ source $(dirname $0)/cluster.sh
 # version will make tests either:
 # 1. Still pass, meaning we can upgrade from earlier than latest release (good).
 # 2. Fail, which might be remedied by bumping this version.
-readonly SERVING_RELEASE_YAML=https://github.com/knative/serving/releases/download/v0.2.1/release-no-mon.yaml
-
-# Cribbed from e2e-tests.sh
-# TODO(#2320): Remove this.
-function publish_test_images() {
-  echo ">> Publishing test images"
-  image_dirs="$(find ${REPO_ROOT_DIR}/test/test_images -mindepth 1 -maxdepth 1 -type d)"
-  for image_dir in ${image_dirs}; do
-    ko publish -P "github.com/knative/serving/test/test_images/$(basename ${image_dir})"
-  done
-}
+readonly LATEST_SERVING_RELEASE_YAML=0.2.2
```

Comment:
Missed this before, I'd rename this `LATEST_SERVING_RELEASE_VERSION`.

Reply:
Good catch. Done.